pkg/cmd/show.go

```diff
@@ -345,7 +345,7 @@ func showKubernetesDashboard(targetReader TargetReader) {
 func showGrafana(targetReader TargetReader) {
 	username, password = getMonitoringCredentials()
 	showPod("grafana", "seed", targetReader)
-	output, err := ExecCmdReturnOutput("bash", "-c", "export KUBECONFIG="+KUBECONFIG+"; kubectl get ingress grafana -n "+getTechnicalID())
+	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "grafana", "-n"+getTechnicalID())
```

Comment:
```suggestion
	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "grafana", "-n", getTechnicalID())
```
your change also works, but I guess we should be more consistent with the arguments (`"-n"+getTechnicalID()` vs `"-n", getTechnicalID()`). Same applies for the other occurences

Reply:
thanks, i also applies this to other occurence

Reply:
You have missed:
> Same applies for the other occurences